build.go

```diff
@@ -4,10 +4,12 @@ import (
 	"container/list"
 
 	"github.com/go-masonry/mortar/interfaces/monitor"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 type promConfig struct {
-	namespace string
+	namespace  string
+	collectors []prometheus.Collector
```

Comment:
I think it's better to rename this to externalCollectors, because these are are only collectors that are created externally of this package.